e2e/cp_test.go

```diff
@@ -4196,3 +4197,44 @@ func TestCountingWriter(t *testing.T) {
 	expected := fs.Expected(t, fs.WithFile(filename, content, fs.WithMode(0644)))
 	assert.Assert(t, fs.Equal(cmd.Dir, expected))
 }
+
+// It should skip special files
+func TestUploadingSocketFile(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	workdir := fs.NewDir(t, t.Name())
+	defer workdir.Remove()
+
+	sockaddr := workdir.Join("/s5cmd.sock")
+	ln, err := net.Listen("unix", sockaddr)
+	if err != nil {
+		t.Fatalf("error occurred while creating a socket file: %v\n", err)
```

Comment:
No need to add new line with `t.Fatal/Fatalf`.
Also you may just `t.Fatal(err)` without context as `go test` would report where it occurred.

Reply:
Resolved. https://github.com/peak/s5cmd/pull/618/commits/c8c642712c1d4b9cbb5daeb2bbfcd3ec5ce39e25